### PR TITLE
Feature/split deviceconfig

### PR DESCRIPTION
**Pull request by suzannemaquelin**
I removed the method __get_organization_plugins from the pluginManager because it seemed to have the same functionality as method get_plugins_of. If not, please comment

### CODE REVIEW

File: source/Configurations/Mock/Light [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "Mock": {
+    "Light": {
+      "module": "plugins.mock.light.Light",
```
Comment (LarsHoldijk) [inline source/Configurations/Mock/Light]: We can leave the collection name and plugin name because it is already due to the folder structure.

**Review by LarsHoldijk — COMMENTED** [review]
I have a few question and possible suggestions for the code. 
 
I don't really think this change solves the complexity of having all configuration in one big file. Could we maybe force the structure that the configuration module has on the plugins module and define the configuration here?

File: source/Configurations/Mock/Light (hunk 1 above)
Comment (szijerveld) [reply source/Configurations/Mock/Light]: I agree

**Review by szijerveld — COMMENTED** [review]
Wouldn't it be better if the configuration files are inside the folder of the plugins?
That way users need only to copy one folder.

File: source/logic/__init__.py [hunk 2]
```diff
@@ -10,7 +8,7 @@
 _device_database = DeviceDatabase("devices")
 
 device_config = get_base_path() + "deviceConfig"
```
Comment (szijerveld) [inline source/logic/__init__.py]: This line is redundant

File: source/pluginmanager/PluginManager.py [hunk 3]
```diff
@@ -39,37 +40,43 @@ def get_plugin(self, organization, plugin_name, required_info):
 
     def get_organizations(self):
         """ Get a list of organizations """
-        organizations = list(self._plugins.keys())
+        plugin_path = get_plugin_path()
+        organizations = list()
+        for o in listdir(plugin_path):
```
Comment (szijerveld) [inline source/pluginmanager/PluginManager.py]: No one char variables

File: source/tests/database_tests/test_device_database_mongo_db.py [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 
+from database.DeviceDatabase import DeviceDatabase
 from models.Device import Device
-from tests import test_util
 
 
 class TestDeviceDatabaseMongoDB(unittest.TestCase):
     def setUp(self):
-        self._database = test_util.get_database()
+        self._database = DeviceDatabase("testing")
```
Comment (szijerveld) [inline source/tests/database_tests/test_device_database_mongo_db.py]: Please use test utils database

**Review by szijerveld — CHANGES_REQUESTED** [review]
First a merge is needed from dev into this branch. 
Also some small remarks

File: source/tests/database_tests/test_device_database_mongo_db.py (hunk 4 above)
Comment (suzannemaquelin) [reply source/tests/database_tests/test_device_database_mongo_db.py]: I deleted test utils since it's only useful for this line

**Review by szijerveld — APPROVED** [review]

**Pull request merged**